content/library/get-started/installation.md

```diff
@@ -156,4 +156,4 @@ Streamlit's officially-supported environment manager for macOS and Linux is [Pip
    streamlit run myfile.py
    ```
 
-3. When you're done using this environment, just type `exit` or press `ctrl-D` to return to your normall shell.
+3. When you're done using this environment, just type `exit` or press `ctrl-D` or `ctrl-C` to return to your normal shell.
```

Comment:
I think this should actually read:

````md
2. Now you can use Python and Streamlit as usual:
   ```sh
   streamlit run myfile.py
   ```
   To stop the Streamlit server, press `ctrl-C`.

3. When you're done using this environment, just type `exit` or press `ctrl-D` to return to your normal shell.
````

Reply:
Fixed on f81fda1.